api/nodes.go

```diff
@@ -506,6 +506,13 @@ type HostVolumeInfo struct {
 	ReadOnly bool
 }
 
+//HostVolumeInfo is used to return metadata about a given HostNetwork
+type HostNetworkInfo struct {
+	Name      string
+	CIDR      string
+	Interface string
```

Comment:
Usually we try to have the api struct match the internal struct pretty closely; is there a reason to exclude the reserved ports in this struct?

Reply:
Oh, and can we add this new object to the [Read Node API docs](https://www.nomadproject.io/api-docs/nodes#read-node) in https://github.com/hashicorp/nomad/blob/main/website/content/api-docs/nodes.mdx as well?

Reply:
> Usually we try to have the api struct match the internal struct pretty closely; is there a reason to exclude the reserved ports in this struct?

Hi @tgross! No, you are right, it was a genuine mistake. 
I am gonna add `ReservedPorts`, update the tests accordingly and also add your test cases for the `<none>`  values following Hashicorp principles... 😉 codify, codify, codify!

> Oh, and can we add this new object to the [Read Node API docs](https://www.nomadproject.io/api-docs/nodes#read-node) in https://github.com/hashicorp/nomad/blob/main/website/content/api-docs/nodes.mdx as well?

Sure, I'll use your example as a template